tools/process_monitor_bpf/process_monitor.c

```diff
@@ -8,6 +8,26 @@
 #include "bpf_helpers.h"
 #include "ebpf_ntos_hooks.h"
 
+#define IMAGE_PATH_SIZE (1024)
+#define COMMAND_SCRATCH_SIZE (32 * 1024) // 32k characters is the max char count that fits in a UNICODE_STRING
```

Comment:
Did we switch this provider over to using UTF16? If we did, do we need to increase this to account for 32K uint16?

Reply:
It's still on UTF-8 for now, and I've been torn about how to feel about that.  If it was UTF-16 that'd be nice for my use cases which often intersect with .NET where strings are natively UTF-16.  But more and more of Windows is embracing UTF-8 so I think that could also remain the right choice.

Do you have opinions on which UTF-* we should be using?  I could swap to UTF-16 if preferred, but I'd like to do that in a subsequent PR.

Reply:
The extension shouldn't be futzing with the string IMHO. It should just pass it through. The only reason I did it was to pack mor characters into the 512 byte stack limit.

Reply:
ok, I'll move to UTF-16 in a followup PR.